src/components/backend-ai-session-list.ts

```diff
@@ -1301,28 +1317,48 @@ export default class BackendAiSessionList extends BackendAIPage {
   configRenderer(root, column?, rowData?) {
     render(
       html`
+        <div class="layout horizontal center flex">
+          <div class="layout horizontal center configuration">
+            ${rowData.item.mounts.length > 0 ? html`
+              <wl-icon class="fg green indicator">folder_open</wl-icon>
+              <button class="mount-button"
+                @mouseenter="${(e) => this._createMountedFolderDropdown(e, rowData.item.mounts)}"
+                @mouseleave="${() => this._removeMountedFolderDropdown()}"
+              >
+                ${rowData.item.mounts.map((item, index, list) => {
+                  if (index + 1 === list.length) {
```

Comment:
if you want to add a single space when the condition doesn't match, then how about using tenary operation?

Reply:
Okay. I'll use it!